static/index.js

```diff
@@ -149,19 +149,30 @@ window.addEventListener('load', e => {
   setupNavigationSection();
 });
 
-const setupBadgeSection = () => {
-  const badgeSection = document.querySelector('#badge_section');
-  const badgeAPI = window.Badge || window.ExperimentalBadge;
+function setBadge(...args) {
+  if (navigator.setExperimentalAppBadge)
+    navigator.setExperimentalAppBadge.apply(navigator, args);
```

Comment:
Nit: Prefer new syntax:

```suggestion
    navigator.setExperimentalAppBadge(...args);
```

Reply:
As above.